healthcareai/tests/test_db_validation.py

```diff
@@ -1,7 +1,7 @@
 import unittest
```

Comment:
- pyodbc error message gone
- file in pull request is called test_db_validation.py but file in pycharm shows test_database_validation.py
- New error message : Exception: "Failed to insert values into [SAM].[dbo].[HCAIPredictionRegressionBASE]. Check that the table exists with right column structure.\n